integration-cli/docker_cli_rmi_test.go

```diff
@@ -342,15 +274,13 @@ RUN echo 2 #layer2
 	// At this point we have 2 containers, one based on layer2 and another based on layer0.
 	// Try to untag "tmp2" without the -f flag.
 	out, _, err = dockerCmdWithError("rmi", newTag)
-	if err == nil || !strings.Contains(out, cid[:12]) || !strings.Contains(out, "(must force)") {
-		c.Log(out)
-		c.Fatalf("%q should not be untagged without the -f flag", newTag)
-	}
+	// should not be untagged without the -f flag
+	c.Assert(err, checker.NotNil)
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?